### PR TITLE
Apply name changes to elastic agent docs

### PR DESCRIPTION
**Pull request by dedemorton**
The focus of this PR is renaming. I'll follow up with additional PRs to fix the steps.

Related PR: https://github.com/elastic/observability-docs/pull/165

NOTE: The docs ci will fail until the companion PR is merged. 

To do (after merging):

- [ ] Rerun ci and merge https://github.com/elastic/observability-docs/pull/165
- [ ] Remove comments added to image include.
- [ ] Set up redirects for changed anchor IDs.

### CODE REVIEW

File: x-pack/elastic-agent/docs/run-elastic-agent.asciidoc [hunk 1]
```diff
@@ -22,14 +22,14 @@ To enroll an {agent} to {fleet}:
...
 . Stop {agent}, if it's already running.
 
-. In {ingest-manager}, click **Settings** and change the defaults, if necessary.
+. In {fleet}, click **Settings** and change the defaults, if necessary.
 For self-managed installations, set the URLs for {es} and {kib}, including
 the http ports, then save your changes.
 +
 [role="screenshot"]
-image::images/kibana-ingest-manager-settings.png[{ingest-manager} settings]
+image::images/kibana-fleet-settings.png[{fleet} settings]
```
Comment (dedemorton) [inline x-pack/elastic-agent/docs/run-elastic-agent.asciidoc:30]:
```suggestion
//image::images/kibana-fleet-settings.png[{fleet} settings]
```

Comment (dedemorton) [reply x-pack/elastic-agent/docs/run-elastic-agent.asciidoc:30]: temporarily commenting out to get a clean build.

**Comment by dedemorton** [comment]
@ph If you review the preview, you should only look at the Elastic Agent docs. I've commented out the image include because it will break the build until the other PR gets merged.

**Review by ph — APPROVED** [review]

**Review by bmorelli25 — APPROVED** [review]

File: x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc [hunk 2]
```diff
@@ -208,8 +208,12 @@ The environment in which the agent will run.
 
 //TODO: Clarify what we mean by environment by showing an example.
 
+// REVIEWERS: Do we need to remove the path settings here? It sounds like
+// we won't allow users to set custom paths because it won't allow them to
+// auto upgrade. Is that correct? 
+
```
Comment (dedemorton) [inline x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc:214]:
```suggestion
```

File: x-pack/elastic-agent/docs/elastic-agent-configuration.asciidoc [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 beta[]
 
+// REVIEWERS: Do we want to use "policy settings" instead of configuration
+// settings in this topic?
+
```
Comment (dedemorton) [inline x-pack/elastic-agent/docs/elastic-agent-configuration.asciidoc:9]:
```suggestion
```

File: x-pack/elastic-agent/docs/running-on-kubernetes.asciidoc [hunk 4]
```diff
@@ -2,6 +2,9 @@
 [role="xpack"]
 === Run {agent} on Kubernetes
 
+//REVIEWERS: This topic is not hooked up to the docs anywhere. Should it be?
+//It needs to be edited.
+
```
Comment (dedemorton) [inline x-pack/elastic-agent/docs/running-on-kubernetes.asciidoc:7]:
```suggestion
```

**Pull request merged**